bokeh/models/plots.py

```diff
@@ -505,6 +505,20 @@ def _check_snapped_toolbar_and_axis(self):
 
     """)
 
+    inner_width = Int(readonly=True, help="""
+    This is the exact width of the plotting canvas, i.e. the width of
+    the actual plot, without toolbars etc. Note this is computed in a
+    web browser, so this property will work only in backends capable of
+    bidirectional communication (server, notebook).
+    """)
+
+    inner_height = Int(readonly=True, help="""
+    This is the exact height of the plotting canvas, i.e. the height of
+    the actual plot, without toolbars etc. Note this is computed in a
+    web browser, so this property will work only in backends capable of
+    bidirectional communication (server, notebook).
+    """)
+
```

Comment:
If it's a quick change to expose frame directly, great, otherwise I'm finally be to proceed with this as a clear improvement